### DISCUSSION TITLE
Setup error

### DISCUSSION THREAD

**Discussion by thundersparkf** [chatwoot/chatwoot#2698] [Q&A] [answered]
<img width="665" alt="Screenshot 2021-07-23 at 7 00 04 PM" src="https://user-images.githubusercontent.com/64605024/126788987-f25f7222-7861-4b2d-84d9-544bc15f0e00.png">
I used the latest image from docker hub and tried the helm Installation. Is something up with my installation or is this a bug? But I tried with the newsletter thing both checked and unchecked.

**Comment by pranavrajs** [comment, marked as answer]
It must be an error with database migration. Can you please verify whether the database setup is complete?
